### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by smdkcc** [mitre/caldera#2833] [Q&A]
Hello, 
I have a problem with Caldera but i don't know how to fix it. When i run caldera, there is this message: 
![image](https://github.com/mitre/caldera/assets/148443576/4069e111-6e8e-4def-93b8-137302d2731d)

**Comment by clenk** [comment]
Hi, were there any other TCP-related errors or warnings above that? And is the `app.contact.tcp` value in your config file set correctly?
